ToDoApp/ToDoApp/ToDoListViewModel.swift

```diff
@@ -39,9 +39,18 @@ final class ToDoListViewModel: ObservableObject {
         }
         fetchToDoTasks()
     }
+    // Tap Edit button
+    func editButtonTapped(todo: TaskModel) {
+        let createTaskViewModel: CreateTaskViewModel = CreateTaskViewModel(taskService: taskService, mode: .editExistingTask(todo)) { [weak self] in self?.resetDestination()
```

Comment:
closure body should come on a new line
```suggestion
        let createTaskViewModel: CreateTaskViewModel = CreateTaskViewModel(taskService: taskService, mode: .editExistingTask(todo)) { [weak self] in 
        self?.resetDestination()
```